frontend/src/domains/places/components/PlaceCard/PlaceCard.tsx

```diff
@@ -11,21 +9,13 @@ import plusIcon from '@/assets/icons/plus.svg';
 import trashIcon from '@/assets/icons/trash.svg';
 import theme from '@/styles/theme';
 
-import deletePlace from '../../apis/deletePlace';
+import { usePlaceCard } from '../../hooks/usePlaceCard';
+import { PlaceBase } from '../../types/place.types';
 import { getCheckedListExcept } from '../../utils/getCheckedListExcept';
 import DatePreviewList from '../DatePreviewList/DatePreviewList';
 import EditPlaceModal from '../EditPlaceModal/EditPlaceModal';
 
-export interface PlaceCardProps {
-  id: number;
-  name: string;
-  address: string;
-  stayDurationMinutes: number;
-  openAt: string;
-  closeAt: string;
-  breakStartAt: string;
-  breakEndAt: string;
-  closedDayOfWeeks: string[];
+export interface PlaceCardProps extends PlaceBase {
```

Comment:
PlaceBase에 있는 브레이크 타임, 체류시간 관련 속성은 PlaceCard에서 사용하지 않을거 같은데 Omit으로 제거하는게 어떨까요??
```suggestion
export interface PlaceCardProps extends PlaceBase Omit<PlaceBase, "stayDurationMinutes" | "breakStartAt" | "breakEndAt"> {
```